src/Activation/ActivationModule.php

```diff
@@ -83,25 +83,6 @@ public function initDb()
     public function handleTranslations(): void
     {
         add_action('core_upgrade_preamble', 'mollieDeleteWPTranslationFiles');
-        add_filter(
-            'site_transient_update_plugins',
-            static function ($value) {
-                if (isset($value->translations)) {
-                    $i = 0;
-                    foreach ($value->translations as $translation) {
-                        if (
-                            $translation["slug"]
-                            === "mollie-payments-for-woocommerce"
-                        ) {
-                            unset($value->translations[$i]);
-                        }
-                        $i++;
-                    }
-                }
-
-                return $value;
-            }
-        );
```

Comment:
What does this relate to?

Reply:
This removes the transients related to the languages. The plugin does not use the .org translations but the local ones, and so on update I removed the files and the transients. Not sure if due to latests changes in wp but now those transients do not seem to have an effect, If i do not remove them and I fix the issue with the new php translations files, I get the local translations, so I removed it.